test/cmdlineTests/metadata_append_standard_json/input.json

```diff
@@ -0,0 +1,33 @@
+{
+	"language": "Solidity",
+	"sources":
+	{
+		"A":
+		{
+			"content": "// SPDX-License-Identifier: GPL-3.0\npragma solidity >=0.0;\n\ncontract test {
+					struct S {
+						uint x;
+					}
+					S s;
+					function f(bool b) public {
+						s.x |= b ? 1 : 2;
+						assert(s.x > 0);
+					}
+				}"
+		}
+	},
+	"settings":
+	{
+		"metadata":
+		{
+			"append": false
+		},
+		"outputSelection":
+		{
+			"A":
+			{
+				"test": ["evm.bytecode", "metadata"]
```

Comment:
TODO: need to remove "metadata" here. Otherwise, tests would likely be flaky.

Reply:
I noticed that `compiler version` will change in `output` for every new release. Maybe in `cmdLineTests.sh` we could fetch the right value and change before the tests are executed ?

Reply:
Fixed this by removing the field `metadata`.

Reply:
Right. `metadata` output is irrelevant to this test so think removing it is the right call.